site/docs/types/type_classes.md

```diff
@@ -66,22 +96,17 @@ name: point
 structure: "NSTRUCT<longitude: i32, latitude: i32>"
 ```
 
-The structure field of a type is only intended to inform systems that don't have built-in support for the type how they can transfer the data type from one point to another without unnecessary serialization/deserialization *and* without loss of type safety. Note that it is currently not possible to "unpack" a user-defined type class into its structure type or components thereof using `FieldReference`s or any other specialized record expression; if support for this is desired for a particular type, this can be accomplished with an extension function.
-
-The structure field is optional. If not specified, the type class is considered to be fully opaque. This implies that a systems without built-in support for the type cannot manipulate values in any way, including moving and cloning. This may be useful for exotic, context-sensitive types, such as raw pointers or identifiers that cannot be cloned.
+The structure field of a type is only intended to inform systems that don't have built-in support for the type about how they can create and transfer values of that type to systems that do support the type.
 
-Note however that the vast majority of types can be trivially moved and copied, even if they cannot be precisely represented using Substrait's built-in types. In this case, it is recommended to use `binary` or `FIXEDBINARY<n>` (where n is the size of the type) as the structure type. For example, an unsigned 32-bit integer type could be defined as follows:
-
-```yaml
-name: u32
-structure: "FIXEDBINARY<4>"
-```
```

Comment:
Based on my understanding, if the intent of the `structure` field is to provide a way for systems that do not support the user-define type to create and transfer values of it, this won't work very well because the systems in questions won't have enough information to create a binary version of that type.

That being said, implicitly if there is no structure and a system without support for a type is given a value of that type, the only thing that system can do is pass the value around as a binary blob.